tree/tree/src/TChain.cxx

```diff
@@ -1373,13 +1375,21 @@ Long64_t TChain::LoadTree(Long64_t entry)
          TIter next(fFriends);
          TFriendLock lock(this, kLoadTree);
          TFriendElement* fe = nullptr;
+         const bool reconstructLocalTreeListOfFriends = fTree->GetListOfFriends() ? false : true;
```

Comment:
This test might not be sufficient.  As the internal tree (within the TChain) may have its own (persistent) friends.

Reply:
If the provided unit test will pass on the CI, maybe we could leave this case for a followup PR? At the moment I don't know about use cases with chain-wide friends and tree-local file-loaded friends at the same time.